scripts/travis/travis_build_and_test.sh

```diff
@@ -7,8 +7,17 @@ MAKE="make --jobs=$NUM_THREADS --keep-going"
 if $WITH_CMAKE; then
   mkdir build
   cd build
-  cmake -DBUILD_python=ON -DCMAKE_BUILD_TYPE=Release -DCPU_ONLY=ON ..
```

Comment:
Can you split this fix into another commit or amend the message to note the fix?